platalea/experiments/flickr8k/basic.py

```diff
@@ -53,4 +53,4 @@
                   )
 
 logging.info('Training')
-result = M.experiment(net, data, run_config)
+result = M.experiment(net, data, run_config, wandb_mode='disabled')
```

Comment:
How to reenable it?
If I set 'wandb online' on the command line, won't the setting be overridden here?

Reply:
`wandb_mode` takes precedence over the command line and the environment variable `WANDB_MODE`. To go back to default behavior, `wandb_mode=None` should be used, which is also the default value. I clarified that in the docstring.

Reply:
I can't find that docstring at the moment. Maybe it's better to have a chat about it this afternoon. Maybe we can even merge it then.